src/Core/Content/Product/SalesChannel/Detail/ProductDetailRoute.php

```diff
@@ -158,8 +158,8 @@ private function findBestVariant(string $productId, SalesChannelContext $context
     {
         $criteria = (new Criteria())
             ->addFilter(new EqualsFilter('product.parentId', $productId))
-            ->addSorting(new FieldSorting('product.price'))
```

Comment:
Thanks for the contribution, looks good to me.

But just one question, the detail route uses a sales channel product repository, which has always a filter `available = true`, so im wondering how can an inactive variant be selected!? 😄 

We will import it and test it. 👍

Reply:
sorry. it was not clear. the variant is available in the frontend but currently out of stock. so it's not buyable but still has the availbale flag for the frontend